src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -140,18 +140,21 @@ function ReportActionItemSingle(props) {
           ]
         : props.action.person;
 
+    const reportID = props.report && props.report.reportID;
```

Comment:
NAB it doesn't seem like the changes to this file are necessary

Reply:
https://github.com/Expensify/App/pull/25906#discussion_r1315741720 So you do want me to revert these changes anyway?

Reply:
Thanks for the link. Let's keep it as is then.